modules/navigation.js

```diff
@@ -0,0 +1,21 @@
+const listSection = document.querySelector('#list');
+const addNewSection = document.querySelector('#add-new');
+const contactSection = document.querySelector('#contact');
+
+export function showListSection() {
+  listSection.style.display = 'block';
+  addNewSection.style.display = 'none';
+  contactSection.style.display = 'none';
+}
+
+export function showAddNew() {
+  addNewSection.style.display = 'block';
+  listSection.style.display = 'none';
+  contactSection.style.display = 'none';
+}
+
+export function showContact() {
+  contactSection.style.display = 'block';
+  addNewSection.style.display = 'none';
+  listSection.style.display = 'none';
```

Comment:
- Please, use arrow syntax for function, It is the practice of the ES6 syntax that you have learned.